pkg/reconciler/inmemorychannel/dispatcher/controller.go

```diff
@@ -80,6 +85,10 @@ func NewController(
 	if err := envconfig.Process("", &env); err != nil {
 		logger.Fatalw("Failed to process env var", zap.Error(err))
 	}
+	kncloudevents.ConfigureConnectionArgs(&kncloudevents.ConnectionArgs{
+		MaxIdleConns:        env.MaxIdleConns,
+		MaxIdleConnsPerHost: env.MaxIdleConnsPerHost,
+	})
```

Comment:
Should we do a quick validation of the values since they are set from env? Will the default zero values be handled properly?